src/query/db_accessor.cpp

```diff
@@ -139,6 +140,8 @@ std::optional<VertexAccessor> SubgraphDbAccessor::FindVertex(storage::Gid gid, s
 
 query::Graph *SubgraphDbAccessor::getGraph() { return graph_; }
 
+storage::StorageMode SubgraphDbAccessor::GetStorageMode() const noexcept { return db_accessor_.GetStorageMode(); }
```

Comment:
If you change this return type to  `auto`, then I think you don't need above include for storage mode

Reply:
Thanks, done!

Reply:
While I like this suggestion, changing return type to `auto` causes these errors across mg_procedure_impl.hpp:
```
error: function 'GetStorageMode' with deduced return type cannot be used before it is defined
```
Given that, I think it’s more prudent to keep this as-is and avoid rearranging other #includes.